plugins/template.md

```diff
@@ -356,6 +356,12 @@ This is a simple <strong>snippet</strong>. Will be replaced: {$somevar1}.
 
 {% include misc/plugin-menu-item-id-boilerplate.md %}
 
+## Example Configuration
+
+This example shows how the template plugin can be used to insert custom templates with pre-defined markup and values.
+
+{% include live-demo.html id="template" height="600" tab="js" %}
```

Comment:
remove the height parameter, it doesn't work that well. The editor will make the room it needs